### PR TITLE
manage gtfs extended modes

### PR DESCRIPTION
**Pull request by nberard**

### CODE REVIEW

File: fixtures/gtfs2ntfs/physical_modes/output/commercial_modes.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Train,Train
+Bus,Bus
+Ferry,Ferry
+CableCar,Cable car
+SuspendedCableCar,Suspended cable car
+Funicular,Funicular
+Coach,Coach
+Air,Airplane
+Taxi,Taxi
+UnknownMode,Unknown mode
```
Comment (datanel) [inline fixtures/gtfs2ntfs/physical_modes/output/commercial_modes.txt]: `Unknown Mode` in the specifications

File: src/test_utils.rs [hunk 2]
```diff
@@ -33,12 +34,20 @@ fn get_file_content<P: AsRef<Path>>(path: P) -> String {
...
+    files_to_check: Option<Vec<&str>>,
     work_dir_expected: &str,
 ) {
     let output_dir = output_dir.as_ref();
-    for filename in files_to_check {
-        let output_file_path = output_dir.join(filename);
+    let files: Vec<String> = match files_to_check {
+        None => fs::read_dir(output_dir)
+            .unwrap()
+            .into_iter()
```
Comment (datanel) [inline src/test_utils.rs]: you can remove this line

File: fixtures/gtfs2ntfs/physical_modes/output/commercial_modes.txt (hunk 1 above)
Comment (nberard) [reply fixtures/gtfs2ntfs/physical_modes/output/commercial_modes.txt]: it's updated in the specs

File: src/gtfs/mod.rs [hunk 3]
```diff
@@ -331,19 +331,25 @@ pub fn read_from_url<P: AsRef<Path>>(
...
-    Other(u16),
+    Coach,
+    Air,
+    Taxi,
+    UnknownMode,
+}
+impl ToString for RouteType {
+    fn to_string(&self) -> String {
+        format!("{:?}", self)
+    }
```
[the remark this answers is not in the document]
Comment (datanel) [reply src/gtfs/mod.rs]: why?

[the remark this answers is not in the document]
Comment (TeXitoi) [reply src/gtfs/mod.rs]: https://doc.rust-lang.org/nightly/std/string/trait.ToString.html

**Pull request merged**